### PR TITLE
fix(version_utils.py): Enabled get Scylla version from private repo

### PR DESCRIPTION
**Pull request by Orenef11**
* Closes: #2430
* Trello: https://trello.com/c/StPEq5j3

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [ ] ~~I added the relevant `backport` labels~~
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: sdcm/utils/version_utils.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 PRIMARY_XML_GZ_REGEX = re.compile(r'="(.*?primary.xml.gz)"')
 
 REPOMD_XML_PATH = "repodata/repomd.xml"
-SCYLLA_URL_REGEX = re.compile(".*(?P<url>http.*scylladb.com.*).*")
+SCYLLA_URL_REGEX = re.compile(r".*(?P<url>http.*(=?scylladb.com|//([0-9]{1,3}\.){3}[0-9]{1,3}).*).*")
```
Comment (fruch) [inline sdcm/utils/version_utils.py:25]: Handling only IP addresses isn't enough, for I'm using something like ngrok and such, I'll have a dns name for it.

```suggestion
SCYLLA_URL_REGEX = re.compile(r".*(?P<url>http?://.*")
```
if it's a valid http, you can follow it up, if there are other case which you don't want to follow, for some reason, put them in the code.

if you thing you care only for specific sections each time, so read the file, and don't use regex for this one. might be easier to handle.

Comment (Orenef11) [reply sdcm/utils/version_utils.py]: I removed this validation

**Comment by eliransin** [comment]
@fruch please backport this to all SCT branches - whithout this we will encounter the same problem when
trying to debug rolling upgrade on versions other than master.

**Comment by roydahan** [comment]
IIUC @eliransin needed that in 2020.1

**Pull request merged**